cpu/esp32/include/periph_cpu.h

```diff
@@ -498,7 +498,7 @@ typedef struct {
 
 
 /**
- * @name   Timer configuration depenend on which implementation is used
+ * @name   Timer configuration depend on which implementation is used
```

Comment:
The intention was to say "Timer configuration depending on which implementation used"

Reply:
Can you please fix that in a follow-up?

Reply:
Sure, but I have no idea yet when I will have time for that. The review process was too fast to react in time. :wink:

Reply:
Sorry about that. Just ACK #15727 and it will be good :)